.github/workflows/ci_image.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         uses: docker/build-push-action@v6
         with:
           push: true
-          tags: hyperledger/iroha2-ci:nightly-2024-09-09
+          tags: hyperledger/iroha2-ci:nightly-2025-05-08
```

Comment:
The Docker image `hyperledger/iroha2-ci:nightly-2025-05-08` does not exist yet; it's a chicken-and-egg problem

Reply:
This time, we have no choice but to ignore these failing checks and merge:

![image](https://github.com/user-attachments/assets/531df97c-f435-4c35-b2d8-475e383b1e49)